client/app/scripts/superdesk-users/users.js

```diff
@@ -1148,6 +1148,10 @@
                         }
                     });
 
+                    scope.isLoggedIn = function(user) {
```

Comment:
This could be moved to the users service since it's used at least 2 times and may be used multiple times in the future.